tests/twitter.test.ts

```diff
@@ -144,6 +144,8 @@ describe("Twitter", () => {
 
   test("Delete Tweet successfully", async () => {
     const logSpy = jest.spyOn(console, "log").mockImplementation(jest.fn());
+    const errorSpy = jest.spyOn(console, "error").mockImplementation(jest.fn());
```

Comment:
This is going to hide all errors. We are just spying it and not expecting errors in the test? If so, we should undo this. having errors is better than hiding all possible errors. Errors will eventually get addressed in a better way if they can't be addressed now.

Reply:
indeed, only spying . expecting errors make the tests brittle and does not make sense to expect the errors because we are mocking the API to return an error

Reply:
On second though, maybe it makes sense to expect the error for some test cases. I will check it again

Reply:
Apologies, it actually make sense to expect the error messages.
Updated the tests to check the errors.